films/models.py

```diff
@@ -0,0 +1,99 @@
+from platform import release
+from django.db import models
+
+# 영화 테이블
```

Comment:
- 불필요한 package 제거
- 불필요한 주석 제거
```suggestion
from django.db import models
```